karma-test-shim.js

```diff
@@ -0,0 +1,55 @@
+Error.stackTraceLimit = 0;
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000;
+
+// (1)
+var builtPaths = (__karma__.config.builtPaths || ['app/'])
+                  .map(function(p) { return '/base/'+p;});
+
+// (2)
+__karma__.loaded = function () { };
+
+// (3)
```

Comment:
In general, comments should make sense to anyone reading the code. I'm not sure why these are numbered - please either explain why, or remove the comments.